research/src/pages/tabs.proposal.mdx

```diff
@@ -0,0 +1,283 @@
+---
+menu: Proposals
+name: Tabs (Editor's Draft)
+path: /components/tabs
+pathToResearch: /components/tabs.research
+showInMenu: false
+---
+
+# Tabs Component Specification
+
+## Overview <a href="#overview" id="overview"></a>
+
+*Tabs* are a set of layered sections of content that display one panel of content at a time. Each tab panel has an associated tab element, that when activated, displays the panel. The list of tab elements is arranged along one edge of the currently displayed panel.
```

Comment:
Tabs are a common mobile pattern that essentially define "sections". Many of the [Material](https://material.io/components/tabs#anatomy) examples show them this way. Therefore I would hesitate to state they are always "layered", as in some mobile UI's (a 2-tab interface), they could slide

Reply:
This is good feedback; there is likely a better word than "layered" which is less design-oriented.

Reply:
I'm going to just remove the word layered for now.